incubator/kafqa/templates/telegraf.configmap.yaml

```diff
@@ -2,7 +2,9 @@ apiVersion: v1
 data:
   telegraf.conf: |-
     [global_tags]
-      cluster_name = {{ .Values.environment | quote }}
+      {{- range $key, $val := .Values.telegraf.labels }}
```

Comment:
Shouldn't the key name be global_tags, instead of labels ? labels is too generic.